src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -217,12 +217,23 @@ def get_crash_type_string(testcase):
                                  CRASH_TYPE_DIMENSION_MAP[crash_type])
 
 
-def filter_stacktrace(stacktrace):
+def filter_stacktrace(stacktrace, blob_name=None, signed_upload_url=None):
   """Filters stacktrace and returns content appropriate for storage as an
   appengine entity."""
   unicode_stacktrace = utils.decode_to_unicode(stacktrace)
   if len(unicode_stacktrace) <= data_types.STACKTRACE_LENGTH_LIMIT:
     return unicode_stacktrace
+  if signed_upload_url:
```

Comment:
Once the migration will signed_upload_url be expected by all callers? Can you leave a comment saying it will be (if it is).